scripts/irods/controller.py

```diff
@@ -293,7 +293,7 @@ def should_return_proc(_p):
 
     if server_proc.is_running():
         try:
-            cur_descendants = filter(should_return_proc, server_proc.children(recursive=True))
+            cur_descendants = list(filter(should_return_proc, server_proc.children(recursive=True)))
```

Comment:
If we're using `difference` below, does that mean `server_descendants` is a `set`? Would it make more sense to use a `set` here to compute `set` difference?

Reply:
`server_descendants` is indeed a `set`. I'll look into usin `set` here instead.

Reply:
Yep, that works. Good catch